roadmaps/configuration.md

```diff
@@ -135,35 +140,43 @@ impact of configuration APIs**
 * Also quantifies action caching inefficiency
 
 <div class="padbottom"></div>
-<span class="etabox">mid 2020</span>**An experimental Bazel mode automatically
+<span class="etabox">Unclear ETA</span>**An experimental Bazel mode automatically
 "trims" build graphs**
 <span class="notstartedstatus">NOT STARTED</span> ([#6524](https://github.com/bazelbuild/bazel/issues/6524))
 
 * e.g. C++ rules don't re-analyze because of Java flag changes
 * Building a binary's *Mac* and *Linux* versions simultaneously doesn't double the build graph
 
 <div class="padbottom"></div>
-<span class="etabox">Jul 2020</span>**An experimental Bazel mode caches
+<span class="etabox">Oct 2020</span>**An experimental Bazel mode caches
 cross-platform Java compilation**
 <span class="inprogressstatus">IN PROGRESS</span> ([#6526](https://github.com/bazelbuild/bazel/issues/6526))
 
 * Improves multiplatform build speed
-* Proof of concepts a [more general
+* Proof-of-concepts a [more general
   design](https://docs.google.com/document/d/17snvmic26-QdGuwVw55Gl0oOufw9sCVuOAvHqGZJFr4/edit)
   that guarantees ideal caching for *all* actions
+* *July update*: This is being distinctly up-prioritized with a ~3 month ETA.
 
 
 ### Features
 
 <div class="padbottom"></div>
-<span class="etabox">Mar 2020</span>**Projects can declare and compliance-check software
```

Comment:
@aiuto how would you suggest we update this?

Reply:
This may or may not happen in Q3.  I expect I'll have it done by late fall. How about saying Q4